alpha_vantage/alphavantage.py

```diff
@@ -271,7 +272,10 @@ def _handle_api_call(self, url):
         if 'json' in self.output_format.lower() or 'pandas' in \
                 self.output_format.lower():
             json_response = response.json()
-            if "Error Message" in json_response:
+            if not json_response:
```

Comment:
I was wondering if we could actually add some unit tests to check all this error cases. There is already a unit tests checking that this function does the job. Would it make sense to add this in here?
```python
    @requests_mock.Mocker()
    def test_handle_api_call(self, mock_request):
        """ Test that api call returns a json file as requested
        """
        av = AlphaVantage(key=TestAlphaVantage._API_KEY_TEST)
        url = "https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY&symbol=MSFT&interval=1min&apikey=test"
        path_file = self.get_file_from_url("mock_time_series")
        with open(path_file) as f:
            mock_request.get(url, text=f.read())
            data = av._handle_api_call(url)
            self.assertIsInstance(
                data, dict, 'Result Data must be a dictionary')
```

Reply:
I think that makes sense to me... I don't think this PR solves the GBTC issue, since it looks like the data just isn't there, but I feel like better error logging is always a good thing.